<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIPerform</class>
 <widget class="QWidget" name="UIPerform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabPatchLevel">
      <attribute name="title">
       <string>Patch/Level</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>778</width>
            <height>242</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="1">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Kbd</string>
             </property>
            </widget>
           </item>
           <item row="0" column="6">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Level</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Patch</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Mute</string>
             </property>
            </widget>
           </item>
           <item row="0" column="7">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Pan</string>
             </property>
            </widget>
           </item>
           <item row="0" column="9">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>RxSw</string>
             </property>
            </widget>
           </item>
           <item row="0" column="8">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>RxCh</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Part</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Group</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Solo</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabOutput">
      <attribute name="title">
       <string>Output</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>778</width>
            <height>242</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QLabel" name="label_43">
             <property name="text">
              <string>Part</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Out</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>MFX#</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Level</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Chorus</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Reverb</string>
             </property>
            </widget>
           </item>
           <item row="0" column="6">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item row="0" column="7">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>2</string>
             </property>
            </widget>
           </item>
           <item row="0" column="8">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item row="0" column="9">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>C</string>
             </property>
            </widget>
           </item>
           <item row="0" column="10">
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>R</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPitch">
      <attribute name="title">
       <string>Pitch</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea_4">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetPitch">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>778</width>
            <height>242</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="0">
            <widget class="QLabel" name="label_44">
             <property name="text">
              <string>Part</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_45">
             <property name="text">
              <string>Oct</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_46">
             <property name="text">
              <string>Crs</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_47">
             <property name="text">
              <string>Fine</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="label_48">
             <property name="text">
              <string>Mono</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QLabel" name="label_49">
             <property name="text">
              <string>Legato</string>
             </property>
            </widget>
           </item>
           <item row="0" column="6">
            <widget class="QLabel" name="label_50">
             <property name="text">
              <string>Bend</string>
             </property>
            </widget>
           </item>
           <item row="0" column="7">
            <widget class="QLabel" name="label_51">
             <property name="text">
              <string>Portam</string>
             </property>
            </widget>
           </item>
           <item row="0" column="8">
            <widget class="QLabel" name="label_52">
             <property name="text">
              <string>Time</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabZone">
      <attribute name="title">
       <string>Zone</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QScrollArea" name="scrollArea_3">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetZone">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>778</width>
            <height>242</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="1">
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>Kbd</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QLabel" name="label_27">
             <property name="text">
              <string>V-Rsv</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>Part</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string>V-Sens</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>Upper</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>Lower</string>
             </property>
            </widget>
           </item>
           <item row="0" column="6">
            <widget class="QLabel" name="label_28">
             <property name="text">
              <string>Oct</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabOffsetVibrato">
      <attribute name="title">
       <string>Offset/Vibrato</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QScrollArea" name="scrollArea_5">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgeOffSetVibrato">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>778</width>
            <height>242</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="0" column="2">
            <widget class="QLabel" name="label_31">
             <property name="text">
              <string>Reso</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_32">
             <property name="text">
              <string>Attack</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="label_33">
             <property name="text">
              <string>Decay</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_29">
             <property name="text">
              <string>Part</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_30">
             <property name="text">
              <string>Cutoff</string>
             </property>
            </widget>
           </item>
           <item row="0" column="7">
            <widget class="QLabel" name="label_36">
             <property name="text">
              <string>Vibr. Depth</string>
             </property>
            </widget>
           </item>
           <item row="0" column="6">
            <widget class="QLabel" name="label_35">
             <property name="text">
              <string>Vibr. Rate</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QLabel" name="label_34">
             <property name="text">
              <string>Release</string>
             </property>
            </widget>
           </item>
           <item row="0" column="8">
            <widget class="QLabel" name="label_37">
             <property name="text">
              <string>Vibr. Delay</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabScale">
      <attribute name="title">
       <string>Scale</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QScrollArea" name="scrollArea_6">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetScale">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>778</width>
            <height>242</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_11">
           <item row="0" column="0">
            <widget class="QLabel" name="label_38">
             <property name="text">
              <string>Part</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_39">
             <property name="text">
              <string>C</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QLabel" name="label_53">
             <property name="text">
              <string>E</string>
             </property>
            </widget>
           </item>
           <item row="0" column="9">
            <widget class="QLabel" name="label_57">
             <property name="text">
              <string>G#</string>
             </property>
            </widget>
           </item>
           <item row="0" column="12">
            <widget class="QLabel" name="label_59">
             <property name="text">
              <string>B</string>
             </property>
            </widget>
           </item>
           <item row="0" column="7">
            <widget class="QLabel" name="label_55">
             <property name="text">
              <string>F#</string>
             </property>
            </widget>
           </item>
           <item row="0" column="6">
            <widget class="QLabel" name="label_54">
             <property name="text">
              <string>F</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_41">
             <property name="text">
              <string>D</string>
             </property>
            </widget>
           </item>
           <item row="0" column="8">
            <widget class="QLabel" name="label_56">
             <property name="text">
              <string>G</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="label_42">
             <property name="text">
              <string>Eb</string>
             </property>
            </widget>
           </item>
           <item row="0" column="10">
            <widget class="QLabel" name="label_58">
             <property name="text">
              <string>A</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_40">
             <property name="text">
              <string>C#</string>
             </property>
            </widget>
           </item>
           <item row="0" column="11">
            <widget class="QLabel" name="label_60">
             <property name="text">
              <string>Bb</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabMIDI">
      <attribute name="title">
       <string>MIDI</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QScrollArea" name="scrollArea_7">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgeMidiSwitches">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>778</width>
            <height>242</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_12">
           <item row="0" column="0">
            <widget class="QLabel" name="label_66">
             <property name="text">
              <string>Part</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_70">
             <property name="toolTip">
              <string>Program change</string>
             </property>
             <property name="text">
              <string>PC</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_72">
             <property name="toolTip">
              <string>Bank select</string>
             </property>
             <property name="text">
              <string>BS</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_67">
             <property name="toolTip">
              <string>Pitch Bend</string>
             </property>
             <property name="text">
              <string>PB</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="label_62">
             <property name="toolTip">
              <string>Polyphonic Aftertouch</string>
             </property>
             <property name="text">
              <string>PA</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QLabel" name="label_68">
             <property name="toolTip">
              <string>Channel Aftertouch</string>
             </property>
             <property name="text">
              <string>CA</string>
             </property>
            </widget>
           </item>
           <item row="0" column="6">
            <widget class="QLabel" name="label_65">
             <property name="toolTip">
              <string>Modulation</string>
             </property>
             <property name="text">
              <string>MD</string>
             </property>
            </widget>
           </item>
           <item row="0" column="7">
            <widget class="QLabel" name="label_69">
             <property name="toolTip">
              <string>Volume</string>
             </property>
             <property name="text">
              <string>VO</string>
             </property>
            </widget>
           </item>
           <item row="0" column="8">
            <widget class="QLabel" name="label_73">
             <property name="toolTip">
              <string>Pan</string>
             </property>
             <property name="text">
              <string>PN</string>
             </property>
            </widget>
           </item>
           <item row="0" column="9">
            <widget class="QLabel" name="label_71">
             <property name="toolTip">
              <string>Expression</string>
             </property>
             <property name="text">
              <string>EX</string>
             </property>
            </widget>
           </item>
           <item row="0" column="10">
            <widget class="QLabel" name="label_63">
             <property name="toolTip">
              <string>Hold 1</string>
             </property>
             <property name="text">
              <string>HD</string>
             </property>
            </widget>
           </item>
           <item row="0" column="11">
            <widget class="QLabel" name="label_64">
             <property name="toolTip">
              <string>Phase lock</string>
             </property>
             <property name="text">
              <string>PL</string>
             </property>
            </widget>
           </item>
           <item row="0" column="12">
            <widget class="QLabel" name="label_61">
             <property name="toolTip">
              <string>Velocity curve</string>
             </property>
             <property name="text">
              <string>VC</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
